app/views/pages/faq.html.erb

```diff
@@ -3,5 +3,140 @@
 <% cache do %>
 <div class="container">
   <h1 class=about_h1>FAQ</h1>
```

Comment:
I wonder if we should spell out FAQ as "Frequently Asked Questions" as the `<h1>` on this page (FAQ elsewhere like in menu bar makes sense).